pubsub/benchmark_test.go

```diff
@@ -57,7 +61,7 @@ func (s *fakeSub) AckFunc() func() { return func() {} }
 // TestReceivePerformance enables characterization of Receive under various
 // situations, characterized in "tests" below.
 func TestReceivePerformance(t *testing.T) {
-	//t.Skip("Skipped by default")
+	t.Skip("Skipped by default")
```

Comment:
You could put it in a separate file with a build tag, too. No preference at this time.

Reply:
Ack.